test/suites/smoke-test-dancelight/test-era-changes.ts

```diff
@@ -0,0 +1,119 @@
+import "@tanssi/api-augment/dancelight";
```

Comment:
Also, why is this tests disabled for starlight?

Reply:
Moved the tests to smoke-test-common-solo in [0d56a85](https://github.com/moondance-labs/tanssi/pull/1119/commits/0d56a852076df19389369ffa0d6a7157e9157282)